xml/Microsoft.VisualC.StlClr/IVector`1.xml

```diff
@@ -472,12 +479,19 @@
         <Parameter Name="_Last_iter" Type="Microsoft.VisualC.StlClr.Generic.ContainerRandomAccessIterator&lt;TValue&gt;" />
       </Parameters>
       <Docs>
-        <param name="">To be added.</param>
-        <param name="_First_iter">To be added.</param>
-        <param name="_Last_iter">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="">An iterator that designates the first element that remains beyond any removed elements, or <see cref="M:Microsoft.VisualC.StlClr.IVector`1.end(Microsoft.VisualC.StlClr.Generic.ContainerRandomAccessIterator{`0}@)" /> if no such element exists.</param>
+        <param name="_First_iter">The beginning of the range of elements to erase.</param>
+        <param name="_Last_iter">The first position beyond the range of elements to erase.</param>
+        <summary>Removes the elements of the controlled sequence in the range \[*\_First\_iter*, *\_Last\_iter*).</summary>
```

Comment:
The start symbol for the range is an opening bracket, the end symbol a closing paren. They should be consistent.